test_regress/t/t_clocking_sched_timing_forkproc.out

```diff
@@ -0,0 +1,5 @@
+%Error: t/t_clocking_sched.v:11:9: Input/output/inout does not appear in port list: 'clk'
+                                 : ... In instance t
+   11 |   input clk;
```

Comment:
I doubt this is really what you want to test - fix the port list.

Reply:
The port list looks fine though. In this case the problem isn't actually directly related to `ForkVisitor`. It occurs when  `V3LinkDot::linkDotPrimary` stage gets re-executed, due to `ForkVisitor` generating new tasks that need to be linked.
Linking them without relying on the entire `linkDotPrimary` pass should fix the issue.